packages/suite/src/components/suite/labeling/MetadataLabeling/MetadataLabeling.tsx

```diff
@@ -169,22 +171,42 @@ const TextLikeLabel = ({
 }: ExtendedProps) => {
     const EditableLabel = useMemo(() => withEditable(RelativeLabel), []);
 
+    let accountTypeUppercase: UppercaseAccountType | undefined;
+
+    if (payload.type === 'accountLabel') {
+        accountTypeUppercase = payload?.accountType.toUpperCase() as UppercaseAccountType;
+    }
+
+    const AccountTypeBadge = payload.type === 'accountLabel' &&
```

Comment:
can you put this component out of this one? and just pass type and account type as prop? Would be nicer to use component instead of `{AccountTypeBadge}`